### ISSUE TITLE
Key repeat doesn't work only with Firefox

### ISSUE THREAD

**Issue by scopedog** [uim/uim#149]
Hi, I just wonder if this can be fixed.

Currently, I'm using uim+mozc on FreeBSD and the key repeat in direct input doesn't work only with Firefox.
The key repeat in Japanese input on Firefox is OK and there is no issue with Thunderbird, Chromium, LibreOffice, xterm, etc.
If I stop uim-xim, Firefox works well.

Could you please investigate this?
If it can be fixed, uim+mozc is perfect for me.

Thank you very much.

**Comment by kou** [comment]
Why do you use uim-xim? You can use uim-gtk3 with Firefox at least on Debian GNU/Linux.

**Comment by scopedog** [comment]
Hi, thank you for your reply.

Unfortunately, FreeBSD's Firefox pkg seems to be built with GTK2 and Firefox won't even start after installing uim-gtk3.

> Why do you use uim-xim?

Do you mean I need to delete 

uim-xim &

from my .xinitrc?

**Comment by kou** [comment]
You can use uim-gtk2 for GTK+ 2 application.

> Do you mean I need to delete

Yes. Normally, you don't need to use XIM.

**Comment by scopedog** [comment]
Thank you.

Unfortunately, none of my programs such as Firefox, Chromium, Thunderbird starts Japanese input without uim-xim though I tried some settings with uim-pref-gtk.
I think very few people use FreeBSD as a desktop OS and there's no tester for the X11 + Japanese environment.

Well, I have decided to switch from Firefox to Chromium because there was also another issue with Firefox. So far everything seems to be OK.
I used to be an X11 programmer and wish I could investigate this a little more deeply but the chance will be low.

I appreciate your help.
